README.rst

```diff
@@ -139,6 +141,10 @@ ev3python.com
     If you are just getting started with programming, we highly recommend
     that you check it out at `ev3python.com`!
 
+Frequently-Asked Questions
```

Comment:
Ah, ok. Github helpfully folded the diff, so that I only saw `Using text-to-speech` above the changes.